### DISCUSSION TITLE
Support displaying/editing multiple difficulties in the osu!taiko editor

### DISCUSSION THREAD

**Discussion by Hiviexd** [ppy/osu#19330] [Ideas]
this is a feature that's currently only present in the osu!mania stable editor. 

it would be an incredibly revolutionary addition to the taiko editor because it would make spread modding, and mapping spreads a much, much better experience.

We currently have to rely on Mapset Verifier to view multiple diffs at once, and rely on 3rd party taiko editors in order to actually edit multiple diffs at once.

Mapset Verifier: 
![image](https://user-images.githubusercontent.com/62819481/180603365-314fd315-80f6-45bd-a104-7d937350ce79.png)

Alchyr's Taiko Editor: https://osu.ppy.sh/community/forums/topics/1373230?n=1
![image](https://user-images.githubusercontent.com/62819481/180603435-1b43861b-55b2-4781-a670-9d4ea3281923.png)

WIP 3rd party taiko editor: 
https://twitter.com/oL0_osu/status/1548544342419451909?s=20&t=Wc5IBfc2IbSJlSPWLwWFOA
https://twitter.com/oL0_osu/status/1526484065611722752?s=20&t=Wc5IBfc2IbSJlSPWLwWFOA

**Comment by TheFearlessDeath500** [comment]
While this is good, wouldn’t that cause some kind of clutter to the mapper? And if osu!mania and osu!taiko have this, why not all rulesets?

**Reply by Rekunan** [reply]
It'd only clutter if the mapper didn't wish for it, and in stable, it's something you have to open up manually (`File` -> `Open Difficulty` -> `For Reference`).
It could be opened up to all rulesets, it just wouldn't be as useful because you'd only be able to look at the timeline, and therefore only compare rhythm choice, which while is the main thing for taiko and mania, std and catch also have an aim component. Unless if the implementation meant that you could view both maps at once, as in say splitting the screen in `1/n` to view `n` maps.

**Comment by Trung0246** [comment]
Since this is Lazer anyways, can this multiple views applicable for mania (with something like horizontal scrolling)?

**Reply by Rekunan** [reply]
If this gets implemented then mania's likely going to also be included since in stable, mania already has this.
